Update ansible-matrix.md

kyliel opened the pull request:
Update the supported modules for azure_module and preview_module according to latest release.

azurecla commented:
Hi __@ChrisCompy__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

ggailey777 commented:
@ChrisCompy this PR now has a merge conflict, possibly because the fix was made in the private repo. If you still want this merged, please fix the conflicts. Otherwise, can you please close it. Thanks!

ggailey777 commented:
@ChrisCompy I am going to close this old PR since it still has a merge conflict and I assume it got fixed in the private repo. If you still need it, reopen and fix the conflict.

kyliel committed:
Update ansible-matrix.md

Update the supported modules for azure_module and preview_module according to latest release.

rloutlaw reviewed (approved):
Looks good!
